---
title: Deliveries presos em "Start as soon as possible", operationMgt in error
description: Descrição
solution: Campaign,Campaign Classic v7,Campaign v8
product: Campaign,Campaign Classic v7,Campaign v8
applies-to: Campaign Classic,Campaign,Campaign Classic v7,Campaign v8
keywords: KCS,Deliveries presos em "Start as soon as possible", operationMgt in error
resolution: Resolution
internal-notes: null
bug: false
article-created-by: Ananya Kuthiala
article-created-date: 5/7/2022 3:45:43 PM
article-published-by: Ananya Kuthiala
article-published-date: 5/7/2022 3:47:08 PM
version-number: 1
article-number: KA-19398
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=d14b53bd-1cce-ec11-a7b5-0022480a8e40
exl-id: 114f9199-b56a-47a3-aea3-ef50d96cdebc
source-git-commit: 6a23faae10364181be7dedc2f408f2ad8d8be3c9
workflow-type: tm+mt
source-wordcount: '116'
ht-degree: 2%

---

# Deliveries presos em &quot;[!UICONTROL Comece o mais rápido possível]&quot;, operationMgt com erro

## Descrição


Os workflows da campanha podem ficar presos em &quot;[!UICONTROL Comece o mais rápido possível]&quot; quando iniciado a partir do [!DNL Campaign] exibir.



Este fluxo de trabalho estava com erro:

```
 scheduler    SCR-160012 JavaScript: error while evaluating script 'operationMgt/scheduler'.
 scheduler    SOP-330011 Error while executing the method 'SubmitJob' of service 'nms:delivery'.
 scheduler   \*\*\*SQL STRING REDACTED FOR CONTAINING CLIENT INFO\*\*\*
 scheduler    PGS-220000 PostgreSQL error: ERROR:  duplicate key value violates unique constraint "nmsdelivery_id" DETAIL:  Key (ideliveryid)=(70806071) already exists.
```

O delivery com Primary key 70806071 (indicado na mensagem de erro) não existia em sua instância


## Resolução


Parecia que o delivery com a chave primária 70806071 existia antes, mas foi excluído.

Ele ainda tinha provas vinculadas a ele (delivery pai). Ao excluir as provas, a variável `operationMgt` O workflow começou a ser executado novamente após uma reinicialização e os workflows começaram a ser executados novamente.
